modules/Link.js

```diff
@@ -81,6 +87,18 @@ class Link extends React.Component {
       this.props
     )
 
+    // If children is a function, we are using a Function as Children Component
+    // so useful values will be passed down to the children function.
+    if(typeof children == 'function'){
```

Comment:
Minor style nitpick: but can you add spaces around the comparison parens? `if (typeof children == 'function') {`